### PR TITLE
add guidelines for evaluations

### PR DESCRIPTION
**Pull request by anaelleltd**
This PR updates the docs with:
- generic guidelines for writing evidences (to be used by members)
- recommendations for assessing the quality of evidence reports (to be used by rank evaluators)

### CODE REVIEW

File: 0000-evidence-template.md [hunk 1]
```diff
@@ -23,23 +23,16 @@
...
 
 
 ## Voting record
-*Provide your voting record in relation to required thresholds.* 
+*Provide your voting record in relation to required thresholds for your rank.* 
 
 |  Ranks | Activity thresholds | Agreement thresholds | Member's voting activities | Comments |
 |---|---|---|---|---|
-|III|70%   |100%  |I have voted on ?? out of ?? referenda in which I was eligible to vote (i.e ?? % voting activity). Out of ?? referenda in which members of higher ranks were in complete agreement, I have voted in line with the consensus ?? times (i.e ?? % voting agreement).  |*This is an example.* |
+|III|70%   |100%  |I have voted on x out of xx referenda in which I was eligible to vote (i.e xx % voting activity). Out of xx referenda in which members of higher ranks were in complete agreement, I have voted in line with the consensus x times (i.e xx % voting agreement).  |*This is an example.* |
```
Comment (alindima) [inline 0000-evidence-template.md:35]: rank 3 is duplicated in this table

File: 0000-evidence-template.md [hunk 2]
```diff
@@ -23,23 +23,16 @@
...
 |I  |90%   |N/A   |   |  |
 |II |80%   |N/A   |   |  |
```
Comment (alindima) [inline 0000-evidence-template.md:37]: ranks 1 and 2 can barely vote on anything, so I don't think it's a valuable metric for these ranks

Comment (athei) [reply 0000-evidence-template.md:37]: Voting activity during the evaluation period is stated as a hard requirement in the manifesto. If we want to remove this from the template the manifesto has to be changed?

File: 0000-evidence-template.md (hunk 1 above)
Comment (anaelleltd) [reply 0000-evidence-template.md:35]: > rank 3 is duplicated in this table

The first line of the table is just an example.

File: 0000-evidence-template.md (hunk 2 above)
Comment (ggwpez) [reply 0000-evidence-template.md:37]: >ranks 1 and 2 can barely vote on anything, so I don't think it's a valuable metric for these ranks

I think it does make sense. The voting activity can still be calculated relative to what they are able to vote on.

**Review by franciscoaguirre — APPROVED** [review]
Looks good

**Pull request merged**